deeplite/tflite_profiler/tflite_profiler.py

```diff
@@ -0,0 +1,78 @@
+import tensorflow as tf
+import tflite
+
+from deeplite.profiler import Profiler, ProfilerFunction
+from deeplite.profiler.utils import Device
+from deeplite.profiler.metrics import Flops
+from deeplite.tf_profiler.tf_data_loader import TFDataLoader, TFForwardPass
+
+
+tf.compat.v1.logging.set_verbosity(tf.compat.v1.logging.ERROR)
+try:
+    tf.compat.v1.enable_eager_execution()
+except:
+    pass
+
+
+class TFLiteProfiler(Profiler):
+    def __init__(self, model, data_splits, **kwargs):
+        super().__init__(model, data_splits, **kwargs)
+        self.backend = 'TFLiteBackend'
+
+    @classmethod
+    def dl_cls(cls):
+        return TFDataLoader
+
+    @classmethod
+    def fp_cls(cls):
+        return TFForwardPass
+
+
+class ComputeFlops(ProfilerFunction):
+    def get_bounded_status_keys(self):
+        return Flops()
+
+    def __call__(self, model, data_splits, batch_size=1, device=Device.CPU, include_weights=True):
+        temp_model = tflite.Model.GetRootAsModel(model, 0)
+
+        with tf.device('gpu' if device == Device.GPU else 'cpu'):
+            return self._compute_flops(temp_model)
+
+    def _compute_flops(self, model):
```

Comment:
Also this function completely ignores any module that isn't Conv2D. Is there an open source library which might provide better coverage of complexity calculations for TFLite? We should support linear layers as well, at a bare minimum

Reply:
I added support for fully connected layers